### ISSUE TITLE
Undesirable indent level of function call arguments

### ISSUE THREAD

**Issue by golopot** [Koihik/LuaFormatter#119]
```
align_args: false
```

The following code shows that the arguments are indented with many extra indent levels. It is undesirable for me because it is not space efficient. Also this kind of indent is unpopular in other languages like javascript or go.

Could we provide an option for the popular way for indent?

Actual:
```lua
aaaaaaaaaaaaaaaaaaaaaa = foo(
                             xxxxxxxxx, yyyyyyyyyyyyyyyyyyy,
                                 zzzzzzzzzzzzzzzzzzzzz
                         )
```

Expected:
```lua
aaaaaaaaaaaaaaaaaaaaaa = foo(
    xxxxxxxxx, yyyyyyyyyyyyyyyyyyy,
    zzzzzzzzzzzzzzzzzzzzz
)
```

**Comment by Koihik** [comment]
Could you provide the whole config file?

**Comment by golopot** [comment]
Full config:
```
keep_simple_control_block_one_line: false
keep_simple_function_one_line: false
break_after_functioncall_lp : true
break_before_functioncall_rp : true
align_args: false
align_parameter: false
chop_down_parameter: true
align_table_field: false
```

**Comment by Koihik** [comment]
Released vscode-lua-format 1.3.4 that fixed this issue.

**Comment by golopot** [comment]
This works in 1.3.4. Thanks.
